### ISSUE TITLE
"Close Settings" button is not localized

### ISSUE THREAD

**Issue by schnubb** [jwplayer/jwplayer#3072]
### Expected Behavior

Aria-Label on the settings close button should be configurable with the localization configuration or would use the localization.close setting.

### Actual Behavior

The aria-label is always "Close Settings".

### Steps to reproduce

1. Load a video with some settings available
2. Open Settings menu and inspect close button
3. Change configuration for `localization.close`
4. redo 2.

### Environment

- Everywhere, cause:
https://github.com/jwplayer/jwplayer/blob/master/src/js/view/controls/components/settings/menu.js#L75

----------------------------------------------------------------------------------------

**Comment by karimJWP** [comment]
Hi @schnubb this is on the road map for our next minor release (8.6.0). If you find more aria-labels that are not being localized, please include them in this ticket
